### PR TITLE
fix: builder shortcuts visibility problem 

### PR DESCRIPTION
**Pull request by raaymax**
Really small but annoying thing

https://github.com/streamsync-cloud/streamsync/assets/649972/2b17fcfb-de2d-48f5-8724-860fceb7f67b

### CODE REVIEW

File: ui/src/builder/BuilderInstanceTracker.vue [hunk 1]
```diff
@@ -45,10 +45,14 @@ const trackElement = (el: HTMLElement) => {
...
 
+	let settingsWidth =
+		document.getElementsByClassName("BuilderSettings")?.[0]?.clientWidth ||
+		0;
+
 	if (!isOffBoundsAllowed.value) {
 		trackerX = Math.max(rendererX, trackerX); // Left boundary
 		trackerX = Math.min(
-			rendererX + rendererWidth - contentsWidth,
+			rendererX + rendererWidth - contentsWidth - settingsWidth,
```
[the remark this answers is not in the document]
Comment (ramedina86) [reply ui/src/builder/BuilderInstanceTracker.vue:55]: metricoff

I tested it using our new PR review apps and yes it's problematic. You can see how far away the Shortcuts are from the selected Metric (in blue).

Please fix, reach out if you need anything. 

If you don't have access to a bigger monitor you can just stretch the window to be wider than the screen.

**Review by ramedina86 — CHANGES_REQUESTED** [review]

File: ui/src/builder/BuilderInstanceTracker.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (raaymax) [reply ui/src/builder/BuilderInstanceTracker.vue]: Fixed

**Comment by ramedina86** [comment]
Tests are failing, strange, looks some invisible characters at play.

**Comment by ramedina86** [comment]
Looking good! Thanks for spotting and fixing this.

I think it's a good example of something that I got to live with without a problem but if someone else points it out, it makes total sense.

**Pull request merged**